segmentstore/storage/src/test/java/io/pravega/segmentstore/storage/chunklayer/ChunkedSegmentStorageMockTests.java

```diff
@@ -345,20 +348,26 @@ public void testFileNotFoundExceptionDuringGarbageCollection() throws Exception
         doThrow(exceptionToThrow).when(spyChunkStorage).doDelete(any());
 
         chunkedSegmentStorage.delete(h1, null).get();
+        Assert.assertEquals(5, chunkedSegmentStorage.getGarbageCollector().getGarbageChunks().size());
+        chunkedSegmentStorage.getGarbageCollector().deleteGarbage(false, 100).get();
         verify(spyChunkStorage, times(5)).doDelete(any());
     }
 
     @Test
     public void testExceptionDuringGarbageCollection() throws Exception {
         String testSegmentName = "test";
         SegmentRollingPolicy policy = new SegmentRollingPolicy(2); // Force rollover after every 2 byte.
-        val config = ChunkedSegmentStorageConfig.DEFAULT_CONFIG.toBuilder().defaultRollingPolicy(policy).build();
+        val config = ChunkedSegmentStorageConfig.DEFAULT_CONFIG.toBuilder()
+                .defaultRollingPolicy(policy)
+                .garbageCollectionFrequencyInSeconds(0)
```

Comment:
Does this mean that your test takes 8s or more to run?

Reply:
No. It is the other way round i want 0 delay.

Reply:
outdated.